.github/workflows/size-limit.yml

```diff
@@ -25,6 +26,18 @@ jobs:
         with:
           node-version: 16
           cache: yarn
+
```

Comment:
Is [the above](https://github.com/Shopify/polaris/blob/ac19525082a944d9d858c5f02470abf82ab065db/.github/workflows/size-limit.yml#L16-L23) needed anymore?

Reply:
Unsure we can investigate in a follow up PR.